> Got this error-
> 
> ```
> # Mutation Summary - Consequences Table
>   ## Navigate to Mutation Summary Page: chr3:g.179218303G>A      ✔ ✔
>   ## Validate Table: Consequences        ✔ ✔ ✔ ✔ ✔ ✘
>         Failed Step: In table "Consequences Mutation Summary" these selections should take the user to correct page in a new tab
>         Specification: specs/gdc_data_portal_v2/mutation_summary/consequences.spec:43
>         Error Message: Timeout 30000.0ms exceeded while waiting for event "page"
>         Stacktrace: 
>         Traceback (most recent call last):
>           File "/Users/manju/Desktop/Manju/GitRepos/gdc-frontend-framework/holmes-py/venv/lib/python3.9/site-packages/getgauge/executor.py", line 33, in execute_method
>             step.impl(*params)
>           File "/Users/manju/Desktop/Manju/GitRepos/gdc-frontend-framework/holmes-py/step_impl/apps/gdc_data_portal_v2/step_defs/generic_steps.py", line 1236, in click_table_by_row_column_check_text_in_new_tab
>             new_tab = APP.shared.click_in_table_handle_new_tab(table_name, v[0], v[1])
>           File "/Users/manju/Desktop/Manju/GitRepos/gdc-frontend-framework/holmes-py/step_impl/base/base_page.py", line 952, in click_in_table_handle_new_tab
>             self.select_specified_table_by_row_column(table_name, row, column)
>           File "/Users/manju/Desktop/Manju/GitRepos/gdc-frontend-framework/holmes-py/venv/lib/python3.9/site-packages/playwright/_impl/_sync_base.py", line 66, in __exit__
>             self._event.value
>           File "/Users/manju/Desktop/Manju/GitRepos/gdc-frontend-framework/holmes-py/venv/lib/python3.9/site-packages/playwright/_impl/_sync_base.py", line 46, in value
>             raise exception
>         playwright._impl._api_types.TimeoutError: Timeout 30000.0ms exceeded while waiting for event "page"
>         
>   ## Navigate back to Mutation Summary Page: chr3:g.179218303G>A         ✔
>   ## Consequences: JSON  ✔ ✔ ✔
>   ## Consequences: TSV   ✔ ✔ ✔
> ```

That's interesting. The external link didn't open on your computer. Can you try two things and tell me what happens:
1. Go to https://portal.gdc.cancer.gov/ssms/a34dbc69-77ad-5c45-9c4b-e26ea62bde83 on non-incognito mode and click on the transcript ENST00000263967 in the consequences table
2. Re-run that automated test 

If it doesn't open correctly for you or Pramod I'll probably have to remove it

--- a/holmes-py/specs/gdc_data_portal_v2/mutation_summary/consequences.spec
+++ b/holmes-py/specs/gdc_data_portal_v2/mutation_summary/consequences.spec
@@ -39,11 +39,6 @@ tags: gdc-data-portal-v2, regression, mutation-summary
     |expected_text                          |row  |column |
     |---------------------------------------|-----|-------|
     |chr3:179148114-179240093 (GRCh38)      |5    |2      |
-* Quick search for "chr3:g.179218303G>A" and go to its page
-* In table "Consequences Mutation Summary" these selections should take the user to correct page in a new tab
-    |row  |column |expected_text                                                        |
-    |-----|-------|---------------------------------------------------------------------|
-    |1    |7      |phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit     |
 
 ## Navigate back to Mutation Summary Page: chr3:g.179218303G>A
 * Quick search for "chr3:g.179218303G>A" and go to its page

